packages/react/package.json

```diff
@@ -69,7 +69,7 @@
     "tslib": "2.4.1"
   },
   "peerDependencies": {
-    "aws-amplify": ">= 5.0.1",
+    "aws-amplify": "^5.0.1",
```

Comment:
Just for my own knowledge, is there a reason we aren't doing all the platform packages?

Reply:
This patch is only targetting react v4 because the commit is the last known good commit for v4. We also only have the changeset for v4 for this reason. For other platform packages we want to upgrade we will need to find a separate commit and make a separate hotfix.

Reply:
Got it ✅